.github/workflows/stlab.yml

```diff
@@ -66,13 +82,34 @@ jobs:
           echo "CC=clang-${{matrix.config.version}}" >> $GITHUB_ENV
           echo "CXX=clang++-${{matrix.config.version}}" >> $GITHUB_ENV
 
-      - name: Configure (Unix)
-        if: ${{ startsWith(matrix.config.os, 'ubuntu') || startsWith(matrix.config.os, 'macos') }}
+      - name: Compile Boost (Ubuntu emscripten)
+        if: ${{ startsWith(matrix.config.compiler, 'emscripten') }}
+        shell: bash
+        run: |
+          mkdir -p ../build-boost
+          emcmake cmake -S $HOME/boost -B ../build-boost -GNinja -DCMAKE_BUILD_TYPE=Release -DCMAKE_CXX_STANDARD=23 \
+            -DCMAKE_CXX_FLAGS="-pthread" \
+            -DCMAKE_INSTALL_PREFIX=`which emsdk`/../upstream/emscripten/cache/sysroot/$HOME/boost-release-emscripten-install \
```

Comment:
I noticed when reading through `Emscripten.cmake` that CMAKE_INSTALL_PREFIX is by default set to blah/cache/sysroot. This line may not actually be needed.

Another thing I noticed in that file is that you can use `em-config CACHE` to get the cache directory in a more direct way.